### DISCUSSION TITLE
How to deploy nextjs project to cpanel?

### DISCUSSION THREAD

**Discussion by pashaskerov21** [vercel/next.js#63129] [App Router]
Next js 14 SSR project deploy to cpanel

**Comment by nikolaikolev196909** [comment]
> Next js 14 SSR project deploy to cpanel

**Comment by shriekdj** [comment]
if your cpanel have support for setup of node.js project then you can use to create a custom server as per docs `https://nextjs.org/docs/pages/building-your-application/configuring/custom-server` which can be used to run your nextjs project.
The cpanel by default supports php but you have to add NodeJS Support if it does not have. Or You have the contact the Hosting Provider for the option of NodeJS app deployement
